content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -173,49 +190,115 @@ The following example shows accessing a service running in the Kubernetes cluste
 1.  First, on the cluster admin machine get the virtual IP address (`clusterIP`) for the service:
 
     {{< text bash >}}
-$ kubectl -n bookinfo get svc productpage -o jsonpath='{.spec.clusterIP}'
-10.55.246.247
+    $ kubectl get svc productpage -o jsonpath='{.spec.clusterIP}'
+    10.55.246.247
     {{< /text >}}
 
 1.  Then on the mesh expansion machine, add the service name and address to its `etc/hosts` file. You can then connect to
     the cluster service from the VM, as in the example below:
 
     {{< text bash >}}
-$ sudo echo "10.55.246.247 productpage.bookinfo.svc.cluster.local" >> /etc/hosts
-$ curl productpage.bookinfo.svc.cluster.local:9080
+$ echo "10.55.246.247 productpage.default.svc.cluster.local" | sudo tee -a /etc/hosts
+$ curl productpage.default.svc.cluster.local:9080
+< HTTP/1.1 200 OK
+< content-type: text/html; charset=utf-8
+< content-length: 1836
+< server: envoy
 ... html content ...
     {{< /text >}}
 
+The `server: envoy` header indicates the traffic indeed is intercepted by sidecar.
+
 ## Running services on a mesh expansion machine
 
-You add VM services to the mesh by configuring a
-[`ServiceEntry`](/docs/reference/config/networking/v1alpha3/service-entry/). A service entry lets you manually add
+1. We first setup a HTTP server on the VM instance, serving HTTP traffic on port 8080.
+
+    {{< text bash >}}
+    $ gcloud compute ssh ${GCE_NAME}
+    $ python -m SimpleHTTPServer 8080
+    {{< /text >}}
+
+1. Determine the GCE instance IP address.
```

Comment:
For GCE specific things, you should say something like: `For example, the command to ... is:`. Otherwise it should say at the top these instructions assume GCE.

Reply:
Good point. Done.